docker/compute_worker/compute_worker.py

```diff
@@ -153,31 +176,45 @@ def _get_bundle(self, url, destination):
         #       :class:`~collections.abc.Mapping`, or an iterable of ``(name, value)``
         #       pairs
 
-
         async with websockets.connect(url) as websocket:
             proc = await asyncio.create_subprocess_exec(
                 *docker_cmd,
                 stdout=asyncio.subprocess.PIPE,
                 stderr=asyncio.subprocess.PIPE
             )
 
+            stdout = b''
+            stderr = b''
+
             while True:
-                data = await proc.stdout.readline()
-                if data:
-                    print("DATA!!!! " + str(data))
-                    await websocket.send(data.decode())
-                else:
+                more_stdout = await proc.stdout.readline()
```

Comment:
maybe make a list of outputs and we can reduce DRY violations here?

Reply:
Not sure I understand what you mean by this?